modules/nw-ingress-setting-a-custom-default-certificate.adoc

```diff
@@ -71,5 +71,24 @@ map[name:custom-certs-default]
 +
 The certificate secret name should match the value used to update the CR.
 
+Your Ingress Controller now uses the custom certificate, but it is not trusted
+by the web console. You must expose your CA certificate so the Ingress
+Controller can communicate with all routes.
+
+. Configure the CA certificate as the cluster proxy certificate:
++
+----
+$ oc -n openshift-config create configmap custom-ca --from-file=ca-bundle.crt=example-ca.crt
+$ oc patch proxy/cluster --type=merge --patch='{"spec":{"trustedCA":{"name":"custom-ca"}}}'
```

Comment:
@codyhoag - @danehans would be the one to review this. It looks like from his comment [1] that there might be different steps depending on whether the cluster-wide proxy is already enabled or not. @danehans Can you confirm?

[1] https://bugzilla.redhat.com/show_bug.cgi?id=1712525#c40